modules/framework/galasa-parent/dev.galasa.framework.k8s.controller/bnd.bnd

```diff
@@ -21,30 +21,30 @@ Import-Package: dev.galasa.framework,\
                 com.google.gson.stream
 Embed-Transitive: true
 Embed-Dependency: *;scope=compile
--includeresource: client-java-16.0.2.jar; lib:=true,\
-    bcpkix-jdk18on-1.79.jar; lib:=true,\
-    bcprov-jdk18on-1.79.jar; lib:=true,\
+-includeresource: client-java-*.jar; lib:=true,\
+    bcpkix-jdk18on-*.jar; lib:=true,\
+    bcprov-jdk18on-*.jar; lib:=true,\
     bcutil-jdk18on-1.79.jar; lib:=true,\
     client-java-api-16.0.2.jar; lib:=true,\
     client-java-proto-16.0.2.jar; lib:=true,\
-    commons-codec-1.16.1.jar; lib:=true,\
-    commons-io-2.16.1.jar; lib:=true,\
-    commons-compress-1.26.0.jar; lib:=true,\
-    commons-lang3-3.14.0.jar; lib:=true,\
-    commons-collections4-4.4.jar; lib:=true,\
+    commons-codec-*.jar; lib:=true,\
+    commons-io-*.jar; lib:=true,\
+    commons-compress-*.jar; lib:=true,\
+    commons-lang3-*.jar; lib:=true,\
+    commons-collections4-*.jar; lib:=true,\
     jose4j-0.7.12.jar; lib:=true,\
     slf4j-api-1.7.36.jar; lib:=true,\
     protobuf-java-3.21.2.jar; lib:=true,\
     jsr305-3.0.2.jar; lib:=true,\
     swagger-annotations-1.6.6.jar; lib:=true,\
```

Comment:
Wondering why not all these other versions are not also captured in the platform ?

Reply:
Task added to the original [story](https://github.com/galasa-dev/projectmanagement/issues/2052) to complete in another change set